### ISSUE TITLE
Accessibility: Incorrect focus order [Add to cart]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#52143]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

The elements on [main section] don't follow a logical (Top to bottom, left to right) keyboard sequence. [Incorrect focus order is being observed, "Continue shopping button" asociated with error message is read first, >> then skip to main content, followed by Header>> Main >> Footer], breaking the expected keyboard navigation for the web page.

![Image](https://github.com/user-attachments/assets/39dfd2b0-1bb1-4321-8fb1-d86aa82c8ceb)

### Component or Page: 

Variation: Add to cart redirect setting

### Name affected element:

1. Continue Shopping button
2. Skip to main content link
3. Header element>> Main content>> Footer

### Relevant code snippets:

`<div class=""wc-block-components-notice-banner__content"">
			<a href=""https://woo.equalizedigital.com/?product=aerodynamic-steel-table"" tabindex=""1"" class=""button wc-forward wp-element-button"">Continue shopping</a> “Aerodynamic Steel Table” has been added to your cart.		</div>`

### Recommended fix:

Change the focus order to match the visual order of elements on the page. Check that the tabindex is not modified and that there is no CSS visually moving elements out of order from how they appear in the DOM.

**Comment by Manussakis** [comment]
This issue doesn't have clear steps to reproduce information 🤔.
How can I see this type of notice message on the Cart page?
However, I believe this issue was fixed with the https://github.com/woocommerce/woocommerce/pull/50061 PR.

cc: @vikrampm1

**Comment by vikrampm1** [comment]
If you're not seeing the issue, I believe it might have been fixed, so I am closing the issue.
